src/theory/bir-support/bir_bool_expScript.sml

```diff
@@ -1,13 +1,12 @@
 open HolKernel Parse boolLib bossLib;
+open bir_auxiliaryLib;
 open bir_auxiliaryTheory;
 open bir_envTheory bir_valuesTheory;
 open bir_immTheory bir_typing_expTheory;
 open bir_exp_memTheory bir_expTheory;
 open bir_exp_immTheory;
 open HolBACoreSimps
 
-open bir_auxiliaryLib;
```

Comment:
Moving the open of `bir_auxiliaryLib` fixed an error. There's some weirdness going here, for sure.